locksmithctl/daemon.go

```diff
@@ -116,6 +117,54 @@ func expBackoff(interval time.Duration) time.Duration {
 	return interval
 }
 
+// getPeriodicAndSleep get the REBOOT_WINDOW_START, REBOOT_WINDOW_LENGTH
+// and check if everything is ok and the get then Periodic. Check if
+// it is outside of the reboot window and If yes then sleep.
+func getPeriodicAndSleep() (err error) {
```

Comment:
Using this notation, you will allocate an emplacement for `err` - but we don't use it since we return `errors.New`. 

```suggestion
func getPeriodicAndSleep() error {
```